app/src/main/java/com/lastaoutdoor/lasta/viewmodel/DiscoverScreenViewModel.kt

```diff
@@ -26,6 +27,15 @@ import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.launch
 
+enum class OrderingBy {
```

Comment:
I think you shouldn't define this here

Reply:
You're right I hadn't thought about the favorite page ! It will indeed probably reuse this feature and I will add it in the utils package.